src/lib/modules/Series/SeriesLiteratureSearch/SeriesLiteratureSearch.js

```diff
@@ -1,5 +1,9 @@
 import { literatureApi } from '@api/literature';
-import { invenioConfig, setReactSearchKitInitialQueryState } from '@config';
+import {
+  invenioConfig,
+  setReactSearchKitInitialQueryState,
+  setReactSearchKitDefaultSortingOnEmptyQueryString,
```

Comment:
Not a showstopper but this variable name reached the 50 characters, maybe replace `ReactSearchKit` with `RSK`

Reply:
it's a long 🐫  :)